packages/web-awesome/src/components/TestResult/TrSteps/TrAttachment.tsx

```diff
@@ -109,7 +109,7 @@ export const TrAttachment: FunctionComponent<{
       {isOpened && isValidComponentType && (
         <div className={styles["test-result-attachment-content-wrapper"]}>
           <div className={styles["test-result-attachment-content"]} role={"button"} onClick={expandAttachment}>
-            <Attachment item={item} />
+            <Attachment item={item} previewable={true} />
```

Comment:
Currently, this option only affects HTML attachments. It embeds the HTML content into the preview area of the attachment step (not in an expanded dialog) instead of displaying its markup.

However, it's better to leave the markup instead of the preview here since not all HTML attachments can be displayed that way (see #226). Also, we have unsolved problems with XSS (#227). Please, revert this change until we solve those issues.